lib/config.ts

```diff
@@ -1,2 +1,21 @@
+/**
+ * @license
+ * Copyright 2017-2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const BALENA_ENGINE_TMP_PATH = '/var/lib/docker/tmp';
```

Comment:
Why this rather than `os.tmpDir`?

Reply:
Good question, I actually meant to ask you about the `tmp` path. `os.tmpDir` would run on the machine where the CLI is running, right? But as the `build-secrets` feature currently requires `balenaEngine`, the relevant tmp path is the one of a device (or VM) running balenaOS. And on such devices, `/tmp` is not not a good tmp path, we actually need something mounted on `/mnt/data` (SD card, read-write permissions, lots of storage), and '/var/lib/docker/tmp' seemed to fit the bill.

Reply:
Makes sense, much better than `os.tmpDir`